### ISSUE TITLE
New language - Hebrew

### ISSUE THREAD

**Issue by thewh1teagle** [picotorrent/picotorrent#1081]
I very love this software. I decided to give a little contribution and translate it into Hebrew.
Here is the link for the translated files - 
https://translate.picotorrent.org/projects/picotorrent/application/he_IL
What are the next steps to add it?

**Comment by StoyanDimitrov** [comment]
The build process will take care of the rest and your contribution will be included in the next release. Meanwhile you can test your work in the non-stable build [here](https://github.com/picotorrent/picotorrent/actions/runs/446911633).

**Comment by thewh1teagle** [comment]
> 
> 
> The build process will take care of the rest and your contribution will be included in the next release. Meanwhile you can test your work in the non-stable build [here](https://github.com/picotorrent/picotorrent/actions/runs/446911633).

Thank you, where can i Download this CI build?

**Comment by thewh1teagle** [comment]
Thank you very much! 
I tried it, and i see that the language is displayed as a LTR language like english, instead of RTL (right to left).
PicoTorrent support RTL languages?

**Comment by vktr** [comment]
I don't think we have explicit support for RTL languages. It seems like we can add support for it though - keeping this open as a feature request for RTL support.
